### PR TITLE
New Dynamic Punch animation

### PR DESCRIPTION
**Pull request by Ridaz**
How it looks: http://i.imgur.com/Bu7qGyU.gif

### CODE REVIEW

File: data/graphics.js [hunk 1]
```diff
@@ -3730,7 +3730,45 @@ var BattleMoveAnims = {
...
+			}, {
+				scale: 4,
+				opacity: 0
+			}, 'linear');
+			battle.showEffect('fireball', {
+				x: defender.x-40,
+				y: defender.y-20,
+				z: defender.z,
+				scale: 0,
+				opacity: .6,
```
Comment (Zarel) [inline data/graphics.js]: Policy is 0.6

**Comment by ascriptmaster** [comment]
Looks like a good start!

It might be nice to darken the background while the animation takes place, but I'll need to see an example of that before I'm sure.

**Comment by Zarel** [comment]
Squash?

**Comment by Ridaz** [comment]
How's this?
![oraora]
(http://i.imgur.com/COxx2EX.gif)

**Pull request merged**